agent/api/task/task.go

```diff
@@ -70,7 +70,7 @@ const (
 
 	NvidiaVisibleDevicesEnvVar = "NVIDIA_VISIBLE_DEVICES"
 	GPUAssociationType         = "gpu"
-	NvidiaRuntime              = "nvidia"
+	NvidiaRuntime              = "ecs-nvidia"
```

Comment:
You may need to make this configurable if we expect people to be able to use the normal Nvidia runtime on other Linux distributions like Ubuntu or Debian.

Reply:
updated